### PR TITLE
Wire to BuildKite, now with branch prefix.

### PR DESCRIPTION
**Pull request by petemounce**
This is #178 but with a branch prefix so you can see it in other CI.

### CODE REVIEW

File: ci/build-test.cmd [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@echo off
+
+"%ProgramFiles%\git\bin\bash.exe" -c "./ci/build-test.sh"
```
Comment (gkassabli) [inline ci/build-test.cmd]: Might be a n00b question, but is this supposed to work on non-Windows laptops?

Comment (jessicafalk) [reply ci/build-test.cmd]: nope, buildkite currently only has windows agents

Comment (gkassabli) [reply ci/build-test.cmd]: Got it. Do we intend to use buildkite for iOS/any other non-Windows builds at some point then?

Comment (jamiebrynes7) [reply ci/build-test.cmd]: Yes - everything will be moved to Buildkite eventually

Comment (jessicafalk) [reply ci/build-test.cmd]: yes, but it's not that easy, so focusing on windows first. but the plan is to completely move to buildkite at some point

Comment (gkassabli) [reply ci/build-test.cmd]: Makes sense

**Comment by jamiebrynes7** [comment]
Superseded by feature/bk-ci

**Pull request closed without merging**